etc/kayobe/kolla.yml

```diff
@@ -251,11 +262,30 @@ kolla_build_blocks:
     {% endif %}
   base_ubuntu_package_sources_list: |
     RUN \
-        rm -f /etc/apt/sources.list && \
+        mv /etc/apt/sources.list /etc/apt/sources.list.backup && \
     {% for repo in stackhpc_ubuntu_jammy_repos %}
         echo '{{ repo }}' >> /etc/apt/sources.list {% if not loop.last %} && \
     {% endif %}
     {% endfor %}
+  footer: |
+    {% if stackhpc_kolla_clean_up_repo_mirrors | bool %}
+    {% if kolla_base_distro == 'centos' %}
```

Comment:
no more centos

Reply:
There is CentOS, we're just not using it :)

Reply:
Good spot. Fixed.